ocfweb/account/templates/account/register/index.html

```diff
@@ -72,8 +91,8 @@
                 <p>
                     Your password should be unique to your OCF account and not
                     shared with any other accounts, including your Berkeley
-                    CalNet account. <em>Never share your password with anyone,
-                    including OCF staff members.</em>
+                    CalNet account. <em>Never share your user account password
+                    with anyone, including OCF staff members.</em>
                 </p>
                 {{form.password|bootstrap}}
                 {{form.verify_password|bootstrap}}
```

Comment:
For group accounts, it would be good to add a wording about using a contact email address that will last between club member generations

Reply:
Sorry, nvm - should have read the entire thread first before code